library/network_connections.py

```diff
@@ -818,12 +818,14 @@ def connection_compare(
             try:
                 con_a.normalize()
             except Exception:
+                # We ignore any errors that might happen
```

Comment:
This comment does not provide any additional value since it just describes the code. The comment should include the reason why it is the right thing to do this here.

Reply:
> This comment does not provide any additional value since it just describes the code. The comment should include the reason why it is the right thing to do this here.

https://github.com/linux-system-roles/network/pull/558#discussion_r1101921414
> The proper comment would be # We ignore any errors that might happen, which seems a pretty meaningless thing to say. This linter rule seems questionable to me, but OK, it needs to be worked around.

We could just say `# Workaround CodeQL warning`, then?